std/process.d

```diff
@@ -3266,8 +3284,7 @@ private struct TestScript
     string path;
 }
 
-version (StdUnittest)
-private string uniqueTempPath() @safe
+package(std) string uniqueTempPath() @safe
```

Comment:
There is also `std.file.deleteme`

Reply:
Good to know, thanks. Not sure we need both - will leave this decision to the future.